### DISCUSSION TITLE
Scaffolding Command

### DISCUSSION THREAD

**Discussion by daceej** [waffle-ops/waffle#93] [Ideas]
I want to add a command to aid with setting up boilerplate code for various things. I don't know what this will look like

Here is the short list of things I'm looking to cover:
- Generate a .travis.yml file or GitHub Actions workflow file for CI
- Set up PHPCS / other code checkers to run in CI
- Possibly CI workflows for automating minor module updates
- A way to install and/or update git hooks (this could probably be a separate command - TBD)
- Add composer.json for Drupal 8/9 using the drupal/core-recommend metapackage
- Add 'default' modules

**Comment by johnnybgoode** [comment]
Adding a couple nice-to-have's to the list:

* A README template
* Boilerplate config for local dev environments (lando, docker, etc)
* A .env file with commonly used vars defined (not necessarily set, unless this is ultimately an interactive process)
* A `custom.settings.local.php` template with the usual overrides, to be appended to the primary `settings.local.php` by `drupal-scaffold`
* `drupal-scaffold` configuration in `composer.json`
